projects/core/src/user/facade/order-return-request.service.spec.ts

```diff
@@ -149,9 +145,7 @@ describe('OrderReturnRequestService', () => {
   });
 
   it('should NOT load order return requests list when user is anonymous', () => {
-    spyOn(userIdService, 'invokeWithUserId').and.callFake((cb) =>
-      cb(OCC_USER_ID_ANONYMOUS)
-    );
+    spyOn(userIdService, 'takeUserId').and.callThrough();
```

Comment:
Why we change the test behaviour? We should in case of anonymous user throw from this spy